misk/src/main/kotlin/misk/config/MiskConfig.kt

```diff
@@ -134,7 +146,7 @@ object MiskConfig {
    */
   fun loadConfigYamlMap(
     appName: String,
-    environment: Environment,
+    environment: String,
```

Comment:
why doesn't this use the `Env` data type?

Reply:
It should. I'll fix.